src/test/commentCounterTest.test.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @jest-environment jsdom
+ */
+
+const commentCounter = require('./commentCounterTest.js');
```

Comment:
- [ ]  Hi, the counter functions here must be those functions that make your App to run(to work). I found out that you have created separate and isolated functions you have used as a test, and this is not what was requested.
- THEREFORE THE REQUIREMENT:
All the counter functions in the App need to be covered with unit tests using Jest. Make sure that you have tests for all edge cases.
* You should think at least about the scenario in which there are 0 items on the page (counter function should return 0)